pkg/store/sqlite/transactions.go

```diff
@@ -131,6 +131,9 @@ func (s *Store) UpdateTransaction(ctx context.Context, t *models.Transaction) (e
 	// Update modified timestamp (in place).
 	t.Modified = time.Now()
 
+	// Update last update timestamp.
+	t.LastUpdate = sql.NullTime{Time: time.Now(), Valid: true}
```

Comment:
This change was made after noticing that the response included a null value for last update.